plugins/snapshots/windows/cimfs.go

```diff
@@ -39,6 +44,13 @@ import (
 	"github.com/containerd/plugin"
 	"github.com/containerd/plugin/registry"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	baseVHDName                           = "blank.vhdx"
+	defaultBaseVhdSizeInBytes      uint64 = 10 * 1024 * 1024 * 1024 // 10 GB
```

Comment:
Rename defaultBaseVhdSizeInBytes to MaxvhdSizeInBytes for better understanding in the context of dynamic vhd creation.